.golangci.yml

```diff
@@ -58,3 +58,5 @@ linters:
         disabled: true
       - name: unused-receiver # disable for now
         disabled: true
+      - name: unhandled-error # disable for now
+        disabled: true
```

Comment:
```suggestion
```
I think this is not needed.

Reply:
Without this, one gets lots of unhandled-errors such as 
```
sda/cmd/api/api_test.go:135:3: unhandled-error: Unhandled error in call to function io.Closer.Close (revive)
		res.Body.Close()
		^
sda/cmd/api/api_test.go:196:3: unhandled-error: Unhandled error in call to function io.Closer.Close (revive)
		res.Body.Close()
		^
sda/cmd/notify/notify_test.go:127:3: unhandled-error: Unhandled error in call to function fmt.Println (revive)
		fmt.Println(err)
```

Reply:
There is a specific check for body-close and similar. Do not blatantly ignore all error returns.

Reply:
But there should not be any need for changes to the linter config.

Reply:
That makes sense. Maybe we should fix them by adding `//nolint:errcheck` to these lines of code? 
I will remove the setting for `unhandled-error`.